subsys/logging/log_backend_net.c

```diff
@@ -57,13 +57,21 @@ struct net_buf_pool *get_data_pool(void)
 static int line_out(u8_t *data, size_t length, void *output_ctx)
 {
 	struct net_context *ctx = (struct net_context *)output_ctx;
+	socklen_t server_addr_len;
 	int ret = -ENOMEM;
 
 	if (ctx == NULL) {
 		return length;
 	}
 
-	ret = net_context_send(ctx, data, length, NULL, K_NO_WAIT, NULL);
+	if (server_addr.sa_family == AF_INET) {
+		server_addr_len = sizeof(struct sockaddr_in);
+	} else {
+		server_addr_len = sizeof(struct sockaddr_in6);
+	}
+
+	ret = net_context_sendto(ctx, data, length, &server_addr,
```

Comment:
We use connected UDP socket thus call net_context_send(), the server address is set in do_net_init(). It was working just fine when I tested it last time.

Reply:
@jukkar Thanks for the feedback. I have adapted the UDP support for eswifi #16930 to work with UDP connection + send as well.

I will close this PR as it's already addressed as part of UDP support.